src/main/java/org/broadinstitute/hellbender/tools/exome/BayesianHetPulldownCalculator.java

```diff
@@ -466,11 +501,11 @@ else if (base == alleleAlt) {
     }
 
     /**
-     * get base counts map from base quality map
+     * Get base counts map from base quality map
      * @param baseQualities map from bases to list of {@link BaseQuality}
      * @return base counts map
      */
-    private Map<Nucleotide, Integer> getBaseCountsFromBaseQualities(final Map<Nucleotide, ArrayList<BaseQuality>> baseQualities) {
+    private Map<Nucleotide, Integer> getBaseCountsFromBaseQualities(final Map<Nucleotide, List<BaseQuality>> baseQualities) {
         Map<Nucleotide, Integer> baseCounts = new HashMap<>();
         for (Nucleotide base : BASES) {
             if (baseQualities.containsKey(base)) {
```

Comment:
Not sure, perhaps this method is used in more than one way, but it seems to me that this if is not necessary since you are making the effort to fill out the map with empty list for non-observed nucleotides in getPileupBaseQualities above. Is it called with maps not produced by that method.?